backend/src/run.py

```diff
@@ -68,71 +67,25 @@ def get(app_instance: Sanic) -> "AppContext":
 app.config.RESPONSE_TIMEOUT = sys.maxsize
 CORS(app)
 
+# Join current file path with the path to the packages folder
+packages_dir = os.path.join(os.path.dirname(__file__), "packages")
+sys.path.append(packages_dir)
```

Comment:
Why? Could we just import e.g. `packages.chaiNNer_standard` and get rid of this? I don't want to mess with python import paths...

Reply:
We're gonna have to do this for plugins anyway right? Since we can't import outside our package. Might as well make sure it works here. (This is what comfyui does btw)

Reply:
But they aren't outside our package and this path is not useful for anything besides our internal packages.

Reply:
Ok

Reply:
I mainly did this for when i was just scanning the dir to import, but since we don't do that anymore it's useless i guess